valuable/tests/enumerable.rs

```diff
@@ -0,0 +1,71 @@
+use valuable::field::*;
+use valuable::*;
+
+#[test]
+fn test_manual_impl() {
```

Comment:
There was no example of implementing Enumerable, so I added a simple manual Enumerable implementation example as a test.